package.json

```diff
@@ -47,7 +47,7 @@
     "moment-timezone": "^0.5.11",
     "node-sass": "^4.4.0",
     "object-unfreeze": "^1.1.0",
-    "pathable-scripts": "git+https://github.com/pathable/pathable-scripts.git",
+    "pathable-scripts": "https://github.com/pathable/pathable-scripts.git",
```

Comment:
`pathable-scripts` are needed here because `p-install` usage and others. The thing is that we should enable the dependency to support to work with `git` protocol, by including `git+https://`. @Dime was facing an issue that his environment was not using `https` and git automatically modified this dependency to use `git` instead. I think we should support it.